### PR TITLE
Update the move command in retrievePreviousBuild to fix bug in windows

### PR DESCRIPTION
**Pull request by zelinh**
### Description
Update the move command in retrievePreviousBuild to fix bug in windows

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by peterzhuamazon — APPROVED** [review]

**Review by prudhvigodithi — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -121,7 +121,7 @@ jacocoTestReport {
     }
 }
 
-String version = '6.2.0'
+String version = '6.2.1'
```
Comment (gaiksaya) [inline build.gradle:124]: Can you not bump the version? I can take care of it in my upcoming PR. Thanks!

Comment (zelinh) [reply build.gradle:124]: When are you planning to bump this version? This is fixing the distribution build so we might want to bump it soon.

Comment (gaiksaya) [reply build.gradle:124]: By EOD. We are having too many tags for 1 fixes. Trying to club 3 changes together.

Comment (zelinh) [reply build.gradle]: I changed it back. Please prioritize the version bump so that we could update our build Jenkinsfiles to start testing. Thanks.

**Pull request merged**